tests/compile/vector_include.pyx

```diff
@@ -0,0 +1,17 @@
+# mode: compile
+# tag: cpp
+
+# Test that using functionality from libcpp.vector does not lead to compile
+# errors when wildcard imports are used as well.
+
+# Import libcpp.vector, which declares PY_SSIZE_T_MAX.
+from libcpp.vector cimport vector
+
+# Import any other module using a wildcard import.
+from spam import *
+
+# Use the imports (details don't matter).
+cdef extern from "vector_include.h":
```

Comment:
```suggestion
cdef extern from *:
    """
    #include <vector>

    std::vector<int> get_vector()
    {
      return std::vector<int>(17);
    }
    """
```

This just lets you get rid of vector_include.h

Reply:
Oh nice, thanks! Updated.